docs/cli/index.md

```diff
@@ -0,0 +1,6 @@
+# forwarder
+
+- [forwarder pac eval](forwarder_pac_eval.md) - Evaluate a PAC file for given URL (or URLs)
+- [forwarder pac server](forwarder_pac_server.md) - Start HTTP server that serves a PAC file
+- [forwarder ready](forwarder_ready.md) - Readiness probe for the Forwarder
+- [forwarder run](forwarder_run.md) - Start HTTP (forward) proxy server
```

Comment:
Could we sort it the way it is in `--help`: 
This is:
1. run
2. pac
3. ready